docs/config.md

```diff
@@ -269,6 +269,34 @@ spec:
     targetCacheTTL: 10
 ```
 
+### JMX Credentials Database
+
+The Cryostat application must be provided with a password to encrypt saved JMX credentials in database. The user can specify the a secret containing the password entry with key `CRYOSTAT_JMX_CREDENTIALS_DB_PASSWORD`. The Cryostat application will use this password to encrypt saved JMX credentials in database.
+
+For example:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: credentials-database-secret
+type: Opaque
+stringData:
+  CRYOSTAT_JMX_CREDENTIALS_DB_PASSWORD: a-very-good-password
+```
+
+Then, the property `.spec.jmxCredentialsDatabaseOptions.databaseSecretName` must be set to use this secret for password.
+
+```yaml
+apiVersion: operator.cryostat.io/v1beta1
+kind: Cryostat
+metadata:
+  name: cryostat-sample
+spec:
+  jmxCredentialsDatabaseOptions:
+    databaseSecretName: credentials-database-secret
+```
+
+**Note**: If the secret is not provided, a default one is generated for this purpose. However, switching between using provided and generated secret is not allowed to avoid password mismatch that causes the Cryostat application's failure to operator the credentials database.
```

Comment:
"failure to access" the credentials database, maybe?

Reply:
ohhh that was a pretty odd type XD let me fix it